src/main/java/net/mcreator/ui/dialogs/DataListSelectorDialog.java

```diff
@@ -62,21 +62,23 @@ public static List<DataListEntry> openMultiSelectorDialog(MCreator mcreator,
 		return dataListSelector.list.getSelectedValuesList();
 	}
 
-	private class DataListCellRenderer extends DefaultListCellRenderer {
+	private static class DataListCellRenderer extends DefaultListCellRenderer {

```

Comment:
Those changes are very unrelated to this PR

Reply:
But I did't change it,maybe this cause the branch change.

Reply:
> Those changes are very unrelated to this PR

But I did't never change it,I only changed the copyright year,maybe you changed the based branch to cause this.